pandas/core/series.py

```diff
@@ -5004,6 +5004,42 @@ def reindex(  # type: ignore[override]
             tolerance=tolerance,
         )
 
+    @overload
+    def rename_axis(  # type: ignore[override]
+        self,
+        mapper: IndexLabel | lib.NoDefault = ...,
+        *,
+        index = ...,
+        axis: Axis = ...,
+        copy: bool = ...,
+        inplace: bool = Literal[True],
```

Comment:
I think you meant

`inplace: Literal[True],` # -> None
`inplace: Literal[False]=...,` # -> Self
`inplace: bool=...,` # -> Self | None

instead of assigning `Literal` as a default argument.

Reply:
Whoops done.